src/rpc/misc.cpp

```diff
@@ -366,6 +368,32 @@ static UniValue setmocktime(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+static UniValue mockscheduler(const JSONRPCRequest& request)
+{
+        RPCHelpMan{"mockscheduler",
```

Comment:
in commit cf04e02bbe14257a1513b029231cb6a21c9766ac:

```suggestion
    RPCHelpMan{"mockscheduler",
```

Excessive whitespace?

Reply:
hmm, I'm seeing the same amount of whitespace on the other `RPCHelpMan` calls in `rpc/misc` & `rpc/net` & `wallet/rpcwallet`. Are you seeing inconsistency? Is there something I'm missing?

Reply:
Feel free to ignore this, but we haven't re-formatted them in b6fb617aaaad5f9cdd7f2ad2825b253ca792055d because they are still in flux.

Reply:
addressed in follow up